### ISSUE TITLE
Laravel package support

### ISSUE THREAD

**Issue by dillingham** [spatie/laravel-ray#10]
While attempting to use `ray()` in a laravel package test suite:

```
"require-dev": {
    "orchestra/testbench": "~5.0",
     "phpunit/phpunit": "^9.2",
    "spatie/laravel-ray": "^1.0"
},
```
```
Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #0 [ <required> array $settings ]] in class Spatie\Ray\Settings\Settings
```

**Comment by freekmurze** [comment]
I'm using it in a package myself and it seems to just work. Could you create an example package to reproduce the issue?

**Comment by dillingham** [comment]
@freekmurze no problem, here you go https://github.com/dillingham/ray-package-example

I just included a test / spatie/laravel-ray & orchestra/testbench. 

```
composer install && ./vendor/bin/phpunit
```

(My apologizes if I am doing something wrong here lol)

**Comment by freekmurze** [comment]
Thanks to your repo I was able to reproduce the issue.

This has been fixed in 1.1.3 of laravel-ray.

You also need to perform this step: https://github.com/spatie/ray/blob/4b3e633a111ab36d6668bb81df64f7efb550f908/docs/getting-started/installation-in-laravel.md#installation-in-an-orchestra-powered-test-suite

Enjoy!

**Comment by dillingham** [comment]
@freekmurze awesome thank you. Works!
